challenges/3doc-liquid-staking/src/interfaces/ISyntheticTokenFactory.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.8.20;
+
+interface ISyntheticTokenFactory {
```

Comment:
Please add methods for the storage variables like `mintedSupply`, `feeReceiver`, etc.